### ISSUE TITLE
Late Notifications

### ISSUE THREAD

**Issue by reyesvicente** [thepracticaldev/dev.to#3372]
**Describe the bug**

There’s a slight delay on notifications. I received a notification that my post passed 1024 views but it already has 1471.
~~**To Reproduce**~~
~~~~

~~~~
~~~~
~~~~
~~~~

**Expected behavior**

I never expected to receive a notification of 1024 views but the view count of the article at the time I received the notification was not 1024.
**Screenshots**

![dev-1](https://user-images.githubusercontent.com/45283313/60562816-b4274c00-9d8b-11e9-9597-186d87314e50.png)
![dev-2](https://user-images.githubusercontent.com/45283313/60562817-b4274c00-9d8b-11e9-80cb-5bf58e17b547.png)

**Desktop (please complete the following information):**

- OS: Mac OS Mojave 10.14.5
- Browser: Brave Browser
- Version: 0.66.99 Chromium: 75.0.3770.100 (Official Build) (64-bit)

~~**Smartphone (please complete the following information):**~~

~~- Device:~~
~~- OS:~~
~~- Browser:~~
~~- Version:~~

**Additional context**
~~

**Comment by rhymes** [comment]
Initially I thought this could have been due to the time that passes between the job being put in the queue and the actual notification.

Instead it's by design, the milestones are set on fixed numbers (some of the notable powers of 2 we use every day in computing), the closest one to the actual number determines the language you'll see in the notification:

https://github.com/thepracticaldev/dev.to/blob/04ce7738abc17861446a53fe0f0038a93ddfbf97/app/services/notifications/milestone/send.rb#L65-L81

Basically, since 1471 it's closer to 1024 than it is to 2048, you get that notification :)

**Comment by reyesvicente** [comment]
> Initially I thought this could have been due to the time that passes between the job being put in the queue and the actual notification.
> 
> Instead it's by design, the milestones are set on fixed numbers (some of the notable powers of 2 we use every day in computing), the closest one to the actual number determines the language you'll see in the notification:
> 
> https://github.com/thepracticaldev/dev.to/blob/04ce7738abc17861446a53fe0f0038a93ddfbf97/app/services/notifications/milestone/send.rb#L65-L81
> 
> Basically, since 1471 it's closer to 1024 than it is to 2048, you get that notification :)

so it isn't real time?

**Comment by rhymes** [comment]
@reyesvicente no
